shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiFhirImplementation.java

```diff
@@ -1,17 +1,21 @@
 package gov.hhs.cdc.trustedintermediary.external.hapi;
 
 import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.fhirpath.IFhirPath;
 import ca.uhn.fhir.parser.IParser;
 import gov.hhs.cdc.trustedintermediary.wrappers.FhirParseException;
 import gov.hhs.cdc.trustedintermediary.wrappers.HapiFhir;
+import java.util.List;
 import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.Base;
+import org.hl7.fhir.r4.model.BooleanType;
 
 /** Concrete implementation that calls the Hapi FHIR library. */
 public class HapiFhirImplementation implements HapiFhir {
 
     private static final HapiFhirImplementation INSTANCE = new HapiFhirImplementation();
-
     private static final FhirContext CONTEXT = FhirContext.forR4();
+    private static final IFhirPath pathEngine = CONTEXT.newFhirPath();
```

Comment:
Make this all caps like the other fields nearby because this is a `static final` field.

```suggestion
    private static final IFhirPath PATH_ENGINE = CONTEXT.newFhirPath();
```

Reply:
Done! Couldn't commit the suggestion cause there were a few other instances